plugins/commands/uninstall_test.go

```diff
@@ -47,14 +47,17 @@ func TestRunUninstallCmd(t *testing.T) {
 	}
 
 	pluginName := filepath.Base(pluginMockPath)
-	pluginExePath := filepath.Join(pluginsDir, utils.GetPluginExecutableName(pluginName))
+	pluginExePath := filepath.Join(pluginsDir, utils.GetLocalPluginExecutableName(pluginName))
 	// Fix path for windows.
 	assert.NoError(t, os.Rename(filepath.Join(pluginsDir, pluginName), pluginExePath))
 
 	// Try uninstalling a plugin that doesn't exist.
 	err = runUninstallCmd("non-existing-plugin")
 	expectedError := generateNoPluginFoundError("non-existing-plugin")
 	assert.Error(t, err)
+	if expectedError == nil {
+		return
```

Comment:
Should this "return" statement be here? If so, worth adding a comment describing why.

Reply:
It is not really needed because the function always returns an error, but the IDE warns about potential NPE.
Added a comment anyway